packages/client/hooks/useAnimatedPhaseListChildren.ts

```diff
@@ -0,0 +1,25 @@
+import {useEffect, useRef, useState} from 'react'
+
+const useAnimatedPhaseListChildren = (isActive: boolean, itemCount: number) => {
+  const [height, setHeight] = useState(isActive ? 'fit-content' : 0)
```

Comment:
made this fit-content so the initial measurement doesn't measure the entire height, just what it needs